MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -964,7 +964,7 @@ protected override string GetCommandLineArguments(string outputPath, EncodingOpt
             if (isEncoding && startNumber > 0)
             {
                 var startTime = state.SegmentLength * startNumber;
-                timeDeltaParam = string.Format("-segment_time_delta -{0}", startTime);
+                timeDeltaParam = string.Format("-segment_time_delta {0}", startTime);
```

Comment:
The more I read (docs and ffmpeg source), the more I'm certain that this is just plain wrong. The recommended time_delta value should be somewhere around `1 / (2 * framerate)`, which in most cases amounts to `0.02` (~24 fps). Can you try changing this as well?

cc @jellyfin/backend

Reply:
yeah, I did try that, by using the original ffmpeg 4.1 patch #639 
However it made things worse. Keeping as such a large number worked fine.

Reply:
I forgot about that patch. Worse how?

Reply:
Actually I'm not sure it was an issue now to thing of it. I had an issue where after some time of playback or jumping around the video, it would just stop playing. But I've since removed the patch to test if the issues still appear and they have, had a couple users mention it, so perhaps thats another issue unrelated to this?

Reply:
That's possible. As long as we don't reintroduce the issues described here https://github.com/jellyfin/jellyfin/pull/664, I think it should be fine.

Reply:
The upside is, this time even if we do reintroduce the same bug, we have the infrastructure (i.e. `jellyfin-ffmpeg`) to fix it for most of the distros. But if it doesn't, we're golden. cc @cvium any other feedback?

Reply:
I still think the value should be fixed. It's clearly being used wrong and removing the minus sign is not enough tbh